tests/README.md

```diff
@@ -10,6 +10,6 @@ The vagrant environment has an unprotected haproxy in front of the apiserver,
 so only `-master` needs to be set to run the tests. 
 
 ```
-cd tests # from the git repo root folder
-go test -master=http://192.168.200.2:8184
```

Comment:
Could you keep the direct invocation example too? Maybe insert it before the vagrant specific section?